### PR TITLE
docs: updates Azure VPC peering service principal

### PR DESCRIPTION
**Pull request by vandjelk**
**Description**

Changes proposed in this pull request:

- Updates Azure VPC peering service principal (SPN).
- Added few formatting changes but there are no changes to the tutorial code.

### CODE REVIEW

File: docs/user/resources/04-40-10-azure-vpc-peering.md [hunk 1]
```diff
@@ -5,20 +5,26 @@ The `azurevpcpeering.cloud-resources.kyma-project.io` custom resource (CR) speci
 Kyma and the remote Azure Virtual Private Cloud (VPC) network. Virtual network peering is only possible within the networks
 of the same cloud provider.
 
-Once an AzureVpcPeering CR is created and reconciled, the Cloud Manager controller first creates a virtual network peering 
-connection in the Virtual Private Cloud (VPC) network of the Kyma cluster in the underlying cloud provider and accepts
-VPC peering connection in the remote cloud provider subscription.
+Once an `AzureVpcPeering` CR is created and reconciled, the Cloud Manager controller creates a VPC peering connection in
+the VPC network of the Kyma cluster in the underlying cloud provider subscription and accepts VPC peering connection in
```
Comment (NHingerl) [inline docs/user/resources/04-40-10-azure-vpc-peering.md:9]:
```suggestion
the VPC network of the Kyma cluster in the underlying cloud provider subscription, and accepts a VPC peering connection in 
```
I think "connection" needs the "a", but let me know if I misunderstand the sentence

File: docs/user/resources/04-40-10-azure-vpc-peering.md [hunk 2]
```diff
@@ -5,20 +5,26 @@ The `azurevpcpeering.cloud-resources.kyma-project.io` custom resource (CR) speci
...
-VPC peering connection in the remote cloud provider subscription.
+Once an `AzureVpcPeering` CR is created and reconciled, the Cloud Manager controller creates a VPC peering connection in
+the VPC network of the Kyma cluster in the underlying cloud provider subscription and accepts VPC peering connection in 
+the remote cloud provider subscription.
 
-You must authorize Cloud Manager service principal `kyma-cloud-manager-ENV` in the remote cloud provider subscription to
-accept VPC peering connection. Assign the following IAM roles to the Cloud Manager service principal in the remote subscription: 
+### Authorization
+
+Cloud Manager must be authorized in the remote cloud provider subscription to accept VPC peering connection.
```
Comment (NHingerl) [inline docs/user/resources/04-40-10-azure-vpc-peering.md:14]:
```suggestion
Cloud Manager must be authorized in the remote cloud provider subscription to accept a VPC peering connection.
```

File: docs/user/resources/04-40-10-azure-vpc-peering.md [hunk 3]
```diff
@@ -5,20 +5,26 @@ The `azurevpcpeering.cloud-resources.kyma-project.io` custom resource (CR) speci
...
 * Network Contributor
 
-AzureVpcPeering can be deleted at any time but the VPC peering connection in the remote subscription must be deleted
-manually.
+**ENV** corresponds to **dev**, **stage**, or **prod**.
 
-## Specification <!-- {docsify-ignore} -->
+### Deleting `AzureVpcPeering`
 
+Kyma's underlying cloud provider VPC peering connection is deleted as a part of AzureVpcPeering deletion. The remote VPC
```
Comment (NHingerl) [inline docs/user/resources/04-40-10-azure-vpc-peering.md:24]:
```suggestion
Kyma's underlying cloud provider VPC peering connection is deleted as a part of the AzureVpcPeering deletion. The remote VPC
```

File: docs/user/resources/04-40-10-azure-vpc-peering.md [hunk 4]
```diff
@@ -5,20 +5,26 @@ The `azurevpcpeering.cloud-resources.kyma-project.io` custom resource (CR) speci
...
+peering connection is left hanging, and must be deleted manually.
```
Comment (NHingerl) [inline docs/user/resources/04-40-10-azure-vpc-peering.md:25]:
```suggestion
peering connection is left hanging, and you must delete it manually.
```

File: docs/user/tutorials/01-30-azure-vpc-peering.md [hunk 5]
```diff
@@ -8,178 +8,173 @@ the steps that create those resources.
...
-   export SUBSCRIPTION_ID=$(az account show --query id -o tsv)
-   export PRINCIPAL_NAME=<Cloud Manager service principal name>
-   export OBJECT_ID=$(az ad sp list --display-name $PRINCIPAL_NAME --query "[].id" -o tsv)
+1.  Log in to Azure and set the active subscription
+    ```shell
+    export SUBSCRIPTION={SUBSCRIPTION}
+    az login
+    az account set --subscription $SUBSCRIPTION
+    ```
+2.  Assign required roles to the Cloud Manager peering service principal
```
Comment (NHingerl) [inline docs/user/tutorials/01-30-azure-vpc-peering.md:17]:
```suggestion
2.  Assign the required roles to the Cloud Manager peering service principal
```

File: docs/user/tutorials/01-30-azure-vpc-peering.md [hunk 6]
```diff
@@ -8,178 +8,173 @@ the steps that create those resources.
...
+            command:
+              - "/bin/bash"
+              - "-c"
+              - "--"
+            args:
+              - "apt update; apt install iputils-ping -y; ping -c 20 $IP_ADDRESS"
+    EOF
+    ```
+
+    This workload should print a sequence of 20 echo replies to stdout.
```
Comment (NHingerl) [inline docs/user/tutorials/01-30-azure-vpc-peering.md:140]: Why "should"? Is this uncertain?
Why not "This workload prints a sequence of 20 echo replies to stdout."
Did something go wrong if it doesn't print 20 echo replies to stdout?

File: docs/user/tutorials/01-30-azure-vpc-peering.md [hunk 7]
```diff
@@ -8,178 +8,173 @@ the steps that create those resources.
...
+
+12. To print the logs of one of the workloads, run:
+
+    ```shell
+    kubectl logs -n $NAMESPACE `kubectl get pod -n $NAMESPACE -l app=azurevpcpeering-demo -o=jsonpath='{.items[0].metadata.name}'`
+    ```
+
+    The command should print an output similar to the following:
```
Comment (NHingerl) [inline docs/user/tutorials/01-30-azure-vpc-peering.md:148]: same question here, can we drop the "should"?

```suggestion
    The command prints an output similar to the following:
```

**Review by NHingerl — COMMENTED** [review]
Up until step 8, you don't use any punctuation at the end of an instruction (with the exception of step 3, which consists of two full sentences.
In the remaining steps, you use some full stops, and sometimes even a "Run:". 

Both approaches are fine, but I'd prefer if it's consistent. Feel free to ignore this, but I'd personally just slap a colon at the end of all instructions that introduce a code snippet (and drop the "Run").

Like 
1. Log in to Azure and set the active subscription:
3. Set the region that is closest to your Kyma cluster. Use `az account list-locations` to list available locations:
10. Create a namespace and export its value as an environment variable:

(Step 12 is fine as is, rephrasing it to "Print the logs of your workloads:" would change the meaning)

File: docs/user/tutorials/01-30-azure-vpc-peering.md (hunk 6 above)
Comment (vandjelk) [reply docs/user/tutorials/01-30-azure-vpc-peering.md:140]: It is more like if VPC peering is in place this command will print a sequence of 20 echo replies to stdout.

I agree that it is nicer without should.

**Review by NHingerl — APPROVED** [review]

**Pull request merged**